### PR TITLE
Replace deprecated procs

### PR DESCRIPTION
**Pull request by Clyybber**

### CODE REVIEW

**Comment by dom96** [comment]
Tests failing :/

**Comment by Clyybber** [comment]
It builds successfully with "nim c /src/nimble" on my machine (arch linux 64bit). Not sure whats going on here...

File: src/nimblepkg/nimscriptsupport.nim [hunk 1]
```diff
@@ -154,7 +154,7 @@ proc setupVM(graph: ModuleGraph; module: PSym; scriptName: string, flags: Flags)
   cbos copyFile:
     os.copyFile(getString(a, 0), getString(a, 1))
   cbos getLastModificationTime:
-    setResult(a, toSeconds(getLastModificationTime(getString(a, 0))))
+    setResult(a, seconds(getLastModificationTime(getString(a, 0))))
```
Comment (GULPF) [inline src/nimblepkg/nimscriptsupport.nim]: This looks wrong,  `toUnix` is the replacement for `toSeconds`

Comment (Clyybber) [reply src/nimblepkg/nimscriptsupport.nim]: ```toUnix```: https://github.com/nim-lang/Nim/blob/master/lib/pure/times.nim#L422

**Comment by Clyybber** [comment]
Thanks @GULPF, tests should now pass.

**Comment by Clyybber** [comment]
@dom96 Tests are passing

**Comment by dom96** [comment]
Thanks!

**Pull request merged**